Firmware/LowLevel/src/main.cpp

```diff
@@ -762,6 +772,25 @@ void loop() {
                 (float) analogRead(PIN_ANALOG_CHARGE_VOLTAGE) * (3.3f / 4096.0f) * ((VIN_R1 + VIN_R2) / VIN_R2);
         status_message.charging_current = llhl_config.options.ignore_charging_current == OptionState::ON ? -1.0f : (float)analogRead(PIN_ANALOG_CHARGE_CURRENT) * (3.3f / 4096.0f) / (CURRENT_SENSE_GAIN * R_SHUNT);
 
+
+        // ESC power saving
+        bool shutdown_escs = false;
+        if (llhl_config.shutdown_esc_max_pitch != 0) {
```

Comment:
@Apehaenger was this the intended usage i.e only enable ESC power saving when the max pitch is set?

Reply:
Sorry I missed the question. Yes exactly, it was indeed reserved for your PR ;-)